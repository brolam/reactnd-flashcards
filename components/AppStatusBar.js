import React from 'react'
import { StyleSheet } from 'react-native'
import { View, StatusBar } from 'react-native'
import { Constants } from 'expo'
import { red } from '../styles/colors'

export default function AppStatusBar() {
  return (
    <View style={styles.container}>
      <StatusBar style={styles.statusBar} barStyle="light-content" />
    </View>
  )
}

const styles = StyleSheet.create({
  container: {
    height: Constants.statusBarHeight,
    backgroundColor: red,
  },
  statusBar: {
    backgroundColor: red,
  }
});