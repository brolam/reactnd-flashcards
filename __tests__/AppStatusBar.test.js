import React from 'react'
import AppStatusBar from '../components/AppStatusBar'

test('renders without crashing', () => {
  const wrapper = shallow(<AppStatusBar />);
  expect(wrapper).toMatchSnapshot();
});