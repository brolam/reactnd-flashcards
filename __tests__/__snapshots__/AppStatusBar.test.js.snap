// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#b71845",
      "height": 44,
    }
  }
>
  <StatusBar
    animated={false}
    barStyle="light-content"
    showHideTransition="fade"
    style={
      Object {
        "backgroundColor": "#b71845",
      }
    }
  />
</View>
`;
